updater/bootstrap.ts

```diff
@@ -182,53 +177,39 @@ async function fetchUpdater(info: IUpdateInfo): Promise<string | null> {
     updaterName = 'preview-updater.exe';
   }
 
-  const reqInfo = {
-    baseUrl: info.baseUrl,
-    uri: `/${updaterName}`,
-  };
   const updaterPath = path.resolve(info.tempDir, updaterName);
   if (await isUpdaterRunning(updaterPath, updaterName)) {
     console.log('Updater is already running, aborting fetch.');
     return null;
   }
 
   const outStream = fs.createWriteStream(updaterPath);
-
   return new Promise((resolve, reject) => {
-    const outPipe = request(reqInfo)
-      .on('response', response => {
-        if (response.statusCode !== 200) {
-          reject(`Failed to fetch updater: status ${response.statusCode}`);
-        }
-      })
-      .pipe(outStream);
-
-    outPipe.on('close', () => {
-      resolve(updaterPath);
-    });
-
-    outPipe.on('error', error => {
-      reject(error);
+    fetch(`${info.baseUrl}/${updaterName}`).then(response => {
+      if (response.status !== 200) {
+        reject(`Failed to fetch updater: status ${response.status}`);
+      }
+      const outPipe = response.body.pipe(outStream);
+      outPipe.on('close', () => {
+        resolve(updaterPath);
+      });
+
+      outPipe.on('error', error => {
+        reject(error);
+      });
     });
   });
 }
 
 async function getLatestVersionInfo(info: IUpdateInfo): Promise<ILatestVersionInfo | null> {
-  const reqInfo = {
-    baseUrl: info.baseUrl,
-    uri: `/${info.versionFileName}`,
-    json: true,
-  };
-
-  const response = await prequest(reqInfo);
-
-  if (response.statusCode !== 200) {
-    console.log(`Failed to fetch version information - ${response.statusCode}`);
+  const response = await fetch(`${info.baseUrl}/${info.versionFileName}`);
 
+  if (response.status !== 200) {
+    console.log(`Failed to fetch version information - ${response.status}`);
     return null;
   }
 
-  return response.body;
```

Comment:
Can you extract all changes to `bootstrap.ts` into a separate PR?  This is the riskiest file in the entire app to change as if it's broken we cannot push new updates.  Any modifications to this file require extremely extensive testing.  I want to do that testing - but I'm not sure we have time to give it the attention it deserves for the next release.